### ISSUE TITLE
[BUG] Issue w/ episode_count (traktSync.db)

### ISSUE THREAD

**Issue by Dealz-416LS** [nixgates/plugin.video.seren#538]
**Logs**
N/A

**Describe the bug**
Recently aired episodes and recently aired episodes marked as watched, the show information is showing a discrepancy which is causing show/widgets to get an issue with indicators.

The Blacklist widget 'Above' is via Seren, 'Below' is via TMDB Helper 
[Widget with missing watched indicator](https://user-images.githubusercontent.com/54026944/109403434-8ece5500-7955-11eb-837f-b3e60c50c00a.png)

`Widget item > (Context menu) > Information`
Seren
[Show infomation - Watched 160 of 159](https://user-images.githubusercontent.com/54026944/109403668-a3f7b380-7956-11eb-82a7-fd249163ea06.png)

TMDB Helper
[Show infomation - Watched 160 of 160](https://user-images.githubusercontent.com/54026944/109403694-e1f4d780-7956-11eb-95d0-1d5cb9342f97.png)

**To Reproduce**
I don't know how to reproduce, the issue is just happening.

**Expected behavior**
1. When all aired episodes are marked as watched, the show should have a watched indicator. 2. When there are newly aired  episodes Show & Widget should have a number to indicate the number of unwatched episode(s).

**Screenshots**
Above

**Kodi Version (please complete the following information):**
 - Kodi Version 18.9 Leia
 - Seren Version 2.0.16

**Additional context**

**Comment by Dealz-416LS** [comment]
Did some more digging on this issue, here is additional information.

 I looked at the `traktSync.db` and below is 2 shows with a newly aired episode, the entries are incorrect.

1# 
`episode_count` should be `149` `unwatched_episodes` should be `1`
[database_issue_1](https://user-images.githubusercontent.com/54026944/110335678-079b7400-801c-11eb-9802-9cdedefc38aa.png)

2# 
`episode_count` should be `24` `unwatched_episodes` should be `1`
[database_issue_2](https://user-images.githubusercontent.com/54026944/110336679-00289a80-801d-11eb-851a-05eb8ac30c83.png)

I assume the data in `traktSync.db` is being pulled from trakt during a sync? judging by the name of the file, I've checked Trakt and it's showing the correct number of total episodes/unwatched (e.g. 148/149) so this data is being saved in the .db incorrectly?

Previously i was wondering where Information (Context Menu) gets it's watched data from, i assume it's from `traktSync.db`?
